code/datums/components/crafting/recipes.dm

```diff
@@ -932,3 +932,13 @@
 	category = CAT_WEAPONRY
 	subcategory = CAT_WEAPON
 	tools = list(TOOL_WIRECUTTER)
+
+/datum/crafting_recipe/fusioncell
+	name = "Fusion Cell"
+	result = /obj/structure/fusioncell
+	time = 30
+	reqs = list(/obj/item/stack/sheet/plasteel = 10,
+				/obj/item/stack/sheet/plasmaglass = 20)
```

Comment:
I feel like it should require a bit more than this, uranium and cells too maybe?